GTG/plugins/send_email/sendEmail.py

```diff
@@ -82,3 +129,26 @@ def onTbTaskButton(self, widget, plugin_api):
 
         Gio.app_info_get_default_for_uri_scheme('mailto').launch_uris(
             ['mailto:' + 'gtg@example.com?' + parameters], None)
+
+    def onTbManagerButton(self, widget, plugin_api):
+        """
+        When the user presses the button in manager.
+        """
+        for tid in self.plugin_api.get_selected():
+            task = self.req.get_task(tid)
+
+        # Body contains Status Tags, Subtasks and Content.
+            body = _("Status: %s") % (task.get_status()) + \
+                _("\nTags: %s") % (", ".join(task.get_tags_name())) + \
+                _("\nSubtasks:\n%s") % (
+                    "\n - ".join([i.get_title() for i in task.get_subtasks()])) + \
+                _("\nTask content:\n%s") % (task.get_excerpt())
+
+            # Title contains the title and the start and due dates.
+            title = _("Task: %(task_title)s") % {'task_title': task.get_title()}
+
+            parameters = urllib.parse.urlencode({'subject': title, 'body': body})
+            parameters = parameters.replace('+', '%20')
+
+            Gio.app_info_get_default_for_uri_scheme('mailto').launch_uris(
+                ['mailto:' + 'gtg@example.com?' + parameters], None)
```

Comment:
there is problem on merging it. as onTbManagerButton handles more than one task while mailing. Instead created a sendEmail function to avoid code repetition.